blog/content/post/2021/03/23/foo.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Running Choria Across the Globe"
+title: "Fighting Latency with Choria"
```

Comment:
Maybe "Fighting Latency" is probably not the best title, since the actual latency increase slightly with this setup… "Reducing overhead", "Faster connections" or something like that might be better or maybe something more "technical" like "Running Choria Leaf Nodes", I don't know what is better :-/

Reply:
`Reducing connection overhead for branch office scenarios` or something like that, thats probably too long - i always struggle with titles too

Reply:
Well, a long and accurate title is better than a short one that is very vague :-)